tests/research/test_b2_05_prereg_freeze.py

```
@@ -623,24 +623,61 @@ def test_outcome_boundary_flags_are_closed():
+    # legitimate future authorized run (BATCH02_DURABLE_EVIDENCE_RETENTION_V1
+    # section 8), so a filesystem-absence check would fail this long-lived
+    # suite the moment a real run ever happens, for a reason unrelated to
+    # prereg integrity. The durable fact this test protects -- no B2-05
+    # result is *committed to source control* -- is instead checked against
+    # Git's tracked-file state, which persistence never touches (the whole
+    # artifacts/ tree is gitignored) and which is exactly what "shipped by
+    # this unit" means.
+    import subprocess
+
+    tracked = subprocess.run(  # noqa: S603 - fixed argv, no shell, test-only
+        ["git", "ls-files", "artifacts/b2_05_flow_absorption"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)